<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="masterProgressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>561</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>361</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QDateEdit" name="daySelector"/>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButtonDownloadForYear">
       <property name="text">
        <string>Download For Year</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QDateEdit" name="yearSelector">
       <property name="displayFormat">
        <string>yyyy</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButtonDownloadForDay">
       <property name="text">
        <string>Download For Day</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QDateEdit" name="monthSelector">
       <property name="displayFormat">
        <string>M/yyyy</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButtonDownloadForMonth">
       <property name="text">
        <string>Download For Month</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNTB_Datacollection_Automation">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuConfigoration">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="openConfFileButton"/>
   </widget>
   <addaction name="menuNTB_Datacollection_Automation"/>
   <addaction name="menuConfigoration"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openConfFileButton">
   <property name="text">
    <string>Open Configuration File</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
